src/dialog_manager.py

```diff
@@ -354,7 +433,8 @@ def __handle_inform(self, prepped_user_input) -> bool:
         # If 1 option left, suggest it, we also cannot ask the user for additional preferences
         if restaurant is not None and other_options is None:
             self.__respond(self.__get_suggestion_string(restaurant))
-            self.__prompt_additional_preferences()
+            if(self.dialog_config["experiment"] == False):
```

Comment:
Just a minor remark, doesn't matter that match, but it's more concise to write this as `if not self.dialog_config["experiment"]:`